R/utils.R

```diff
@@ -17,6 +19,39 @@ var_labels_for <- function(df, vars) {
   render_safe(unname(formatters::var_labels(df, fill = TRUE)[vars]))
 }
 
+
+#' Helper function to convert to months if needed
+#'
+#' @param x (`numeric`) time.
+#' @param unit (`character`) or (`factor`) time unit.
+#'
+#' @returns A `numeric` vector with the time in months.
+#'
+#' @export
+convert_to_month <- function(x, unit) {
```

Comment:
hi @BFalquet , I was wondering if you could add test for this?

Reply:
actually we have some tests (https://github.com/insightsengineering/chevron/blob/b8bfb33a83f8cd0fe0f09dad6f34eab37f334bd0/tests/testthat/test-utils.R#L206) but since I didn't move them, they don't appear in the modified files. The function was just moved to improve the file organization, sorry for the confusion